changelog/platform_dependent_execinfo.dd

```diff
@@ -0,0 +1,25 @@
+Platform dependent execinfo introspection added
+
+A new module (core.internal.execinfo) has been added for platform dependent
+execinfo detection. On every POSIX system which provides an execinfo
+implementation as part of its C runtime the appropriate implementation-
+dependent execinfo module will be imported automatically.
+
+Besides that, there is an opportunity for using external execinfo
+implementations built as separated libraries and linking druntime with them.
+In that case exactly one of the following version ID must be set for druntime
```

Comment:
Could you clarify the wording to make it obvious that this version ID needs to be set when Druntime is compiled ?
That means, if one package it for Musl, one has to choose whether it depends on libexecinfo or not, and the end user cannot change this.